### PR TITLE
feat: @badger-dao/sdk-mocks package

### PR DESCRIPTION
**Pull request by NamelsKing**

### CODE REVIEW

**Comment by NamelsKing** [comment]
image
Hook worked properly

File: mocks.pkg/.env.exmpl [hunk 1]
```diff
@@ -0,0 +1,7 @@
+  ETHEREUM_RPC_NODE=
+  POLYGON_RPC_NODE=
+  ARBITRUM_RPC_NODE=
+  BINANCE_RPC_NODE=
+  AVALANCHE_RPC_NODE=
+  FANROM_RPC_NODE=
+
```
Comment (axejintao) [inline mocks.pkg/.env.exmpl:7]: nit if blockers
rename `env.example` remove extra newline

File: mocks.pkg/README.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
+<div> 
+  <img align="right" src="docs/images/badger.png" height="150" />
+</div>
+
+# Badger SDK Mocks
```
Comment (axejintao) [inline mocks.pkg/README.md:5]: any reason to choose `mocks.pkg` vs. `mocks`?

File: mocks.pkg/__cache__/methods.json [hunk 3]
```diff
@@ -0,0 +1,51 @@
+{
+  "digg": {
```
Comment (axejintao) [inline mocks.pkg/__cache__/methods.json:2]: 🚀 🚀 🚀

File: mocks.pkg/roadmap [hunk 4]
```diff
@@ -0,0 +1,10 @@
+- fill methods args cfgs for chains (eth)
+- ask jintao to cr8 a package for sdk
+- add rpc nods url to ENV
+- add generation and deploy in github actions
+
+improvments:
```
Comment (axejintao) [inline mocks.pkg/roadmap:6]: do we want to make an md or docs for this one or remove it

**Review by axejintao — APPROVED** [review]

File: mocks.pkg/.env.exmpl (hunk 1 above)
Comment (NamelsKing) [reply mocks.pkg/.env.exmpl:7]: kk

File: mocks.pkg/roadmap (hunk 4 above)
Comment (NamelsKing) [reply mocks.pkg/roadmap:6]: ye, it`s a quick notes, i`ll transfer all of that

**Pull request merged**